server/datastore/mysql/migrations/tables/20240703193943_AddBundleIdentifierToSoftwareTitles.go

```diff
@@ -0,0 +1,92 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240703193943, Down_20240703193943)
+}
+
+func Up_20240703193943(tx *sql.Tx) error {
+	if columnExists(tx, "software_titles", "bundle_identifier") {
```

Comment:
Just for my own understanding, is that a check we should always add when adding columns? Did we run into issues in the past where that was not done?

Reply:
full context is: around the time I was writing this migration, you and Robert were dealing with some migrations that went out of order in Dogfood.

I noticed this (old!) function and I figured it might save us a headache if we don't run the migration if the table is already there, given how complex it is.

no other good reason, so I'm totally ok to remove it too.

Reply:
Makes sense! That's fine for me, was just curious.